Sources/PackageModel/Manifest/PackageDependencyDescription.swift

```diff
@@ -68,15 +67,15 @@ public struct PackageDependencyDescription: Equatable, Codable {
         }
 
         self.explicitName = name
-        self.name = name ?? PackageReference.computeDefaultName(fromURL: normalizedURL)
+        self.name = name ?? PackageIdentity(normalizedURL).computedName
```

Comment:
should we carry the PackageIdentity here in addition (or better - instead) to the name?

Reply:
Why do we need to store a computed name here at all?  Is it for later by-name lookups?

Reply:
@mattt wdyt?

Reply:
This is out of date. The current version looks like this:

```swift
  /// The name of the package, if available.
    public var name: String {
        self._name ?? self._identity.computedName
    }

    /// Create a package reference given its identity and repository.
    public init(identity: Identity, path: String, name: String? = nil, kind: Kind = .remote) {
        self._identity = PackageIdentity(identity)
        self._name = name
        self.path = path
        self.kind = kind
    }
```